src/templates/newengland-organizing.html

```diff
@@ -0,0 +1,102 @@
+                <p>Matthew McMillan designs certificates using GIMP for competitions he organize since he has some graphic design skills. If you’d like him to do this for your competition, please let him know well in advance so he has time to do this. Oh, by the way, when you do get in touch with us, let us know what your favorite flavor of ice cream is. That lets us know you’re actually reading this document. (Yes, we’re serious).</p>
+                <h3 id="sponsorship">Sponsorship</h3>
+                <p>Many competitions are sponsored by cube stores, who will provide prizes and sometimes even vend at the competition. You should try to figure this out before announcing.</p>
+                <h3 id="cubingusa-supported-competitions">CubingUSA Supported Competitions</h3>
+                <p>CubingUSA is supporting some competitions. Here’s a brief summary from their website:</p>
+                <blockquote>
+                    <ul>
+                        <li><em>Organizers must submit an application including an estimated budget at least 60 days in advance.</em></li>
+
+                        <li><em>Any net income goes into the CubingUSA general fund.</em></li>
+
+                        <li><em>Organizers must agree to the CubingUSA Code of Conduct as seen on the Documents page.</em></li>
+
+                        <li><em>Notably, organizers may not personally financially benefit and must report any potential conflicts of interest.</em></li>
+
+                        <li><em>Expenses must be pre-approved and may only be reimbursed with receipts.</em></li>
+
+                        <li><em>Expenses that were not pre-approved may be rejected at CubingUSA’s discretion.</em></li>
+
+                        <li><em>All sponsorships, partnerships, and prizes must be pre-approved.</em></li>
+
+                        <li><em>A CubingUSA Deputy, appointed by the CubingUSA Board of Directors, must be involved in the organization of the competition and must be present at the competition to ensure relevant policies are followed. (CubingUSA Deputies may or may not also be an organizer or WCA Delegate.)</em></li>
+
+                        <li><em>A Financial Report must be submitted after the competition.</em></li>
+                    </ul>
+                </blockquote>
+                <p>– <em><a href="https://cubingusa.org/supported">CubingUSA</a></em></p>
+                <p>The main benefit of this is that you can use their nonprofit status (which can result in venue discounts) and that finances are handled in a more official way. If this is something you’re interested in, let us and we can work to make this happen. Note that this doesn’t change what the competition is actually like the day of.</p>
+                <h3 id="coordinating-staff">Coordinating Staff</h3>
```

Comment:
This header has capitalized words, others don't.  Be consistent (probably capitalize everywhere)